packages/grafana-ui/src/components/GraphNG/GraphNG.tsx

```diff
@@ -33,6 +33,7 @@ export interface GraphNGProps extends Omit<PlotProps, 'data' | 'config'> {
   legend?: LegendOptions;
   fields?: XYFieldMatchers; // default will assume timeseries data
   onLegendClick?: (event: GraphNGLegendEvent) => void;
+  onSeriesColorChange?: (label: string, color: string) => void;
```

Comment:
is `label: string` our best identifier?  I get a bit nervous about checking equality based on the display names that get changed through the override process :grimacing:

Reply:
Since this is get's saved as an override, I think this is our only identifier that will work across query refreshes.

Reply:
@ryantxu yeah, for now I think this is what we have to go with. But as I stated in the special notes above it would be better to do some refactoring and use the `dataFrameFieldIndex` as the identifier.

But as @torkelo highlighted, we probably still need to use the label in the fieldOverride to is will work across query refreshes.